.github/workflows/integration_tests.yml

```diff
@@ -3,7 +3,7 @@ name: Integration Testing
 on:
   push:
     branches: [ "main" ]
-  pull_request:
+  pull_request_target:
```

Comment:
I'm wondering if we should do like adapters: https://github.com/dbt-labs/dbt-snowflake/blob/main/.github/workflows/integration.yml#L33-L34 and don't specify a branch. From what I understand, by specifying `branches: ["main"]` - external contribs that target main branch will automatically trigger CI. Maybe we want the ability to manually run CI like adapters team.



```
pull_request_target:
```